www.foia.gov/_sass/modules/_agency_preview.scss

```diff
@@ -1,24 +1,65 @@
-.agency-preview_components {
-  list-style: none;
-}
-
-.agency-preview_list-hed {
-  @extend %h2;
-}
-
-.agency-preview_component {
+%link-caret {
   color: $color-primary-darkest;
   cursor: pointer;
-  border-bottom: 1px solid $color-primary-alt;
+  margin-right: 1em; // Make room for the caret
   position: relative;
 
   &:after {
-    bottom: -3px;
-    content: '>';
+    bottom: -.2em;
+    content: '>'; // TODO this should be a glyph
     display: inline-block;
-    font-size: 16px;
     font-weight: 900;
     position: absolute;
-    right: -1.2rem;
+    right: -.8em;
+  }
+}
+
+.agency-preview {
+  margin-top: $space-4x;
+
+  a {
+    @extend %a-colors;
+  }
+
+  h2 {
+    font-family: $font-serif;
+    font-size: $h3-font-size;
+    font-weight: $font-bold;
+  }
+
+  h3 {
+    font-size: $h2-font-size;
+    margin-bottom: $space-5x;
+    margin-top: $space-1x;
+  }
+
+  h4 {
+    font-family: $font-sans;
+    font-size: $base-font-size;
   }
 }
+
+.agency-preview_agency-back {
+  @extend %link-caret;
+
+  &:after {
+    bottom: -.1em;
+  }
+}
+
+.agency-preview_components {
+  list-style: none;
```

Comment:
There's also a `usa-unstyled-list` class, if this is the only thing this needs to style.

Reply:
ha, it is now, but originally had more styles before it was finalized.

Reply:
unfortunately, usa-unstyled-list also reduces the spacing between li's.